iina/MiniPlayerWindowController.swift

```diff
@@ -395,8 +395,8 @@ class MiniPlayerWindowController: NSWindowController, NSWindowDelegate, NSPopove
   func updateVideoSize() {
     guard let window = window else { return }
     let videoView = player.mainWindow.videoView
-    let (width, height) = player.videoSizeForDisplay
-    let aspect = CGFloat(width) / CGFloat(height)
+    let (width, height) = player.originalVideoSize
+    let aspect = width == 0 || height == 0 ? 1 : CGFloat(width) / CGFloat(height)
```

Comment:
Very poor readability, consider adding parentheses around `width == 0 || height == 0`. Or write a helper function (extension?) since I see you use it mutiple times